pom.xml

```diff
@@ -21,7 +21,8 @@
   <properties>
     <hadoop.version>2.7.2</hadoop.version>
     <hive.version>2.3.4</hive.version>
-    <jdk.version>1.7</jdk.version>
+    <jdk.version>1.8</jdk.version>
```

Comment:
This is a major, backwards incompatible change so we need a major version bump. I think the plan was that we were going to do a 2.0.0 release with what's in master now and then another release with the extensions? If so let's do that, then make this a 3.0.0 release. We'll also need the CHANGELOG.md updated to call out this JDK version change and then also a line describing that the extensions for JUnit5 are now available.